### PR TITLE
Add missing parenthesis in microbenchmarks

### PR DESCRIPTION
**Pull request by auscompgeek**
Fixes: #923

### CODE REVIEW

File: tests/microbenchmarks.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def test_dict_set(test_case):
             dict["meow"] = 4
             dict["cockadoodledoo"] = 5
             dict["hiss"] = 6
-    """, timed=True)
+    """), timed=True)
 
 def test_class_init(test_case):
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E302) expected 2 blank lines, found 1

File: tests/microbenchmarks.py [hunk 2]
```diff
@@ -203,7 +203,7 @@ def test_dict_set(test_case):
...
     print("Running" , "test_class_init")
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 20: (E203) whitespace before ','

**Comment by auscompgeek** [comment]
Hmm. From the looks of things, this entire file doesn't follow PEP 8 whitespace. Should I reformat the entire file, or just leave it as is?

**Comment by freakboy3742** [comment]
@auscompgeek Oh, *please* reformat it :-) 

It appears that the pre-commit linting check wasn't working for a while, so a lot of problems have drifted in over time. Unfortunately, this now means that a lot of people are getting tagged with problems that aren't actually theirs. If you're able to fix them, that would be most appreciated.

**Comment by auscompgeek** [comment]
Ok. I'd probably want to run an autoformatter in that case. I am seeing some sort of vague style preference in this file that black and yapf trample over though — both black and yapf want to put the `adjust` calls on separate lines.

black:

```diff
--- tests/microbenchmarks.py    2018-09-22 03:03:08.239819 +0000
+++ tests/microbenchmarks.py    2018-09-22 03:03:19.308244 +0000
@@ -1,24 +1,34 @@
 import sys
 from os import path
+
 sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
 
 from utils import *
+
 
 def test_small_integers(test_case):
     print("Running", "test_small_integers")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust(
+            """
         for i in range(100):
             for j in range(100):
                 x = -5
                 for k in range(-5, 257):
                     x = x + 1
-        """), timed=True)
+        """
+        ),
+        timed=True,
+    )
+
 
 def test_booleans(test_case):
     print("Running", "test_booleans")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust(
+            """
         for i in range(100000):
             x = True
             y = False
 
             x = (not x and y)
```

yapf (default config):
```diff
--- tests/microbenchmarks.py    (original)
+++ tests/microbenchmarks.py    (reformatted)
@@ -4,19 +4,24 @@
 
 from utils import *
 
+
 def test_small_integers(test_case):
     print("Running", "test_small_integers")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
         for i in range(100):
             for j in range(100):
                 x = -5
                 for k in range(-5, 257):
                     x = x + 1
-        """), timed=True)
+        """),
+        timed=True)
+
 
 def test_booleans(test_case):
     print("Running", "test_booleans")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
         for i in range(100000):
             x = True
             y = False
```

Now, usually I'm not a fan of autopep8, but would you rather just make pycodestyle happy, or should we go for what yapf is suggesting?

**Comment by freakboy3742** [comment]
@auscompgeek The immediate requirement is just to make pycodestyle happy; I've been contemplating a project-wide move to black because it seems to be the way things are moving. I can't claim I'm in love with 100% of black's settings, but I can't say I've seen anything I *actively* hate either, and having an opinionated and automatable set of defaults is vastly preferable to vague styles.

**Pull request merged**